fiftyone/server/routes/embeddings_v2.py

```
@@ -75,45 +78,43 @@ def get_sample_filter(slices):
     async def post(self, request: Request, data: dict) -> dict:
-        """Lists the dataset's visualization runs (cheap; run docs only)."""
+        """Cheap ready/error peek at the dataset's visualization runs.
+
+        A poll target: one raw collection read keyed by the dataset's ID
+        (the client already has it), no ``Dataset`` object (schema
+        resolution, sample reload) and no results loading — safe to hit
+        on an interval while a run is pending.
+        """
         return await run_sync_task(self._post_sync, data)
 
     def _post_sync(self, data):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_post_sync`

(ANN202)